src/rust/dbv2/migrations/2024-01-09-160715_price_to_nominal/up.sql

```diff
@@ -0,0 +1,35 @@
+-- Your SQL goes here
+CREATE FUNCTION integer_price_to_quote_indivisible_subunits(market_id numeric, price numeric) RETURNS NUMERIC IMMUTABLE AS $$
```

Comment:
Since we already merged #672 I don't think this file should be here any more

Maybe you should delete this migration then merge `main` into this branch?